package/pyproject.toml

```
@@ -95,13 +95,27 @@ doc = [
     "pybtex-docutils",
 ]
 
-
 [project.urls]
-Documentation = 'https://docs.mdanalysis.org/'
-"User Guide" = 'https://userguide.mdanalysis.org/'
-"Issue Tracker" = 'https://github.com/mdanalysis/mdanalysis/issues'
-"User Group" = 'https://groups.google.com/g/mdnalysis-discussion/'
-Discord = 'https://discord.com/channels/807348386012987462/'
-Blog = 'https://www.mdanalysis.org/blog/'
+"Project URL" = "https://www.mdanalysis.org"
+"Download URL" = "https://github.com/MDAnalysis/mdanalysis/releases"
```

```suggestion
Homepage = "https://www.mdanalysis.org"
Downloads = "https://github.com/MDAnalysis/mdanalysis/releases"
```
? Just a suggestion, but "URL" feels a repetition since the field is already `[project.urls]`.

---

Looks good, I went with this but with a non-plural Download instead.